client/app/components/Link.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Button, { ButtonProps } from "antd/lib/button";
+
+type AnchorButtonProps = React.AnchorHTMLAttributes<any> & React.ButtonHTMLAttributes<any>;
+
+// TODO: adapt click handlers to keypress events
+function DefaultLinkComponent({ type, role, ...props }: AnchorButtonProps) {
+  return role === "button" ? (
+    <button type="button" style={{ all: "unset" }} {...props} />
+  ) : (
+    <a role={role} type={type} {...props} /> // eslint-disable-line jsx-a11y/anchor-has-content
```

Comment:
Maybe explicitly pull out the `children` attribute so we don't have to disable the linter and make it more explicit what's happening?